orchagent/vxlanorch.cpp

```diff
@@ -731,6 +802,26 @@ bool VxlanTunnelMapOrch::addOperation(const Request& request)
     SWSS_LOG_NOTICE("Vxlan tunnel map entry '%s' for tunnel '%s' was created",
                    tunnel_map_entry_name.c_str(), tunnel_name.c_str());
 
+    //ip link add <vxlan_dev_name> type vxlan id <vni> local <src_ip> remote <dst_ip>  dstport 4789
```

Comment:
This must go to VxlanMgr. In fact, handling the ConfigDB entry for L2 Vxlan is already moved to VxlanMgr.

Reply:
At the time we wrote L2 vxlan code vxlanMgrd was not there. Sure, we will move these codes to vxlanMgrd.

Reply:
we has refine code according to your suggestion.